src/client/datascience/jupyter/interpreter/nbconvertInterpreterDependencyChecker.ts

```diff
@@ -21,12 +21,15 @@ export class NbConvertInterpreterDependencyChecker implements INbConvertInterpre
         @inject(IJupyterCommandFactory) private readonly commandFactory: IJupyterCommandFactory
     ) {}
 
-    // Check to see if nbconvert is installed in the given interpreter
+    // Check to see if nbconvert is installed in the given interpreter, note that we also need jupyter since that supplies the needed
+    // template files for conversion
     public async isNbConvertInstalled(interpreter: PythonEnvironment, _token?: CancellationToken): Promise<boolean> {
         if (this.nbconvertInstalledInInterpreter.has(interpreter.path)) {
             return true;
         }
-        const isInstalled: boolean = !!(await this.installer.isInstalled(Product.nbconvert, interpreter));
+        const isInstalled: boolean =
```

Comment:
Our dependency installer always lays down jupyter + nbconvert so we didn't often get into this situation. But this catches the case where nbconvert is installed into an env without jupyter.